Vizzuality/half-earth-v3
# Feature/species card miscellaneous updates

## Species cards feedback updates
### Description
This is a miscellaneous PR addressing some of the feedback items received in [this basecamp thread](https://vizzuality.atlassian.net/browse/HE-173), namely:
- Contain images on species widget
- Title and warning on species card.
- Tooltip explaining terrestrial vertebrates.
- WDPA default layer.
- Remove flagship and endangered species filter.
- Link species common name to MoL species page.
### Testing instructions
From the data globe click on a precalculated area and check the updates on the sidebar.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-173
https://vizzuality.atlassian.net/browse/HE-178
https://vizzuality.atlassian.net/browse/HE-183
https://vizzuality.atlassian.net/browse/HE-184
https://vizzuality.atlassian.net/browse/HE-186
https://vizzuality.atlassian.net/browse/HE-187
